sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityFunctionType.java

```diff
@@ -17,7 +17,17 @@
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public enum SensitivityFunctionType {
-    BRANCH_ACTIVE_POWER, // MW
-    BRANCH_CURRENT, // A
+    /**
+     * @deprecated Use {@link #BRANCH_ACTIVE_POWER_1} instead.
```

Comment:
But we didnt' handle json migration?
So when reading a olf factor json, we should shift it to BRANCH_ACTIVE_POWER_1 and BRANCH_CURRENT_1 ?

Reply:
I can swap them in serializer and deserializer.